pubsub/gcppubsub/gcppubsub_test.go

```diff
@@ -32,27 +32,7 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-const (
-	// These constants capture values that were used during the last -record.
-	//
-	// If you want to use --record mode,
-	// 1a. Create a topic in your GCP project:
-	//    https://console.cloud.google.com/cloudpubsub, then
-	//    "Enable API", "Create a topic".
-	// 1b. Create a subscription by clicking on the topic, then clicking on
-	//    the icon at the top with a "Create subscription" tooltip.
-	// 1c. Create a second subscription the same way.
-	// 2. Update the topicName constant to your topic name, and the
-	//    subscriptionName0 and subscriptionName1 constants to your
-	//    subscription names.
-	topicName         = "test-topic"
-	subscriptionName0 = "test-subscription-1"
-	subscriptionName1 = "test-subscription-2"
-	projectID         = "go-cloud-test-216917"
-
-	benchmarkTopicName        = "benchmark-topic"
-	benchmarkSubscriptionName = "benchmark-subscription"
-)
+const projectID = "go-cloud-test-216917"
```

Comment:
Can you restore some of the comments for this constant, e.g. "This constant captures the projectID that was used during the last --record." ?

Reply:
Done.